### ISSUE TITLE
Fully remove "rule.id" as URL feature

### ISSUE THREAD

**Issue by jesusgn90** [wazuh/wazuh-kibana-app#1584]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 3.9.3   | 7.2     | 520 |

**Description**
The code was not removed properly, right now, there is a piece of code that keeps some of the related logic still doing things before rendering the table visualizations / Discover.

**Steps to reproduce**

1. Go to Discover.
2. Click on some "rule.id" field.
3. You'll see a broken link.

**Screenshots**

![image](https://user-images.githubusercontent.com/8860227/61110996-d0f90900-a488-11e9-8730-e8b7dff5a21e.png)

**Comment by jesusgn90** [comment]
For 3.9.3 we've decided to fully remove the `rule.id` as URL feature (pending restore it properly in further releases). For now, I've found that some parts of the code were not fully removed.

https://github.com/wazuh/wazuh-kibana-app/commit/5a9663310070725c84f76451673a34d4bfcfc2dc - Here we removed all the pending code from this feature.

https://github.com/wazuh/wazuh-kibana-app/commit/50d2c9c3805cbc295ada738e54113828e4056475 - And here, we removed it from the _fieldsFormatMap_ in the index pattern.

**Pending testing**
